packages/picasso-lab/src/Dropzone/story/Default.example.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useState } from 'react'
+import {
+  Dropzone,
+  DropzoneErrorCode,
+  DropzoneFileUpload,
+  DropzoneFileError,
+  DropzoneFileRejection,
+  DropzoneDropEvent
+} from '@toptal/picasso-lab'
+
+type ValidatorType = (
+  file: File
+) => DropzoneFileError | DropzoneFileError[] | null
+type OnDropType = (
+  acceptedFiles: File[],
+  fileRejections: DropzoneFileRejection[],
+  event: DropzoneDropEvent
+) => void
+
+const MAX_SIZE = 600 * 1000
+
+const customSizeValidator: ValidatorType = file => {
+  if (file.size > MAX_SIZE) {
+    return {
+      code: DropzoneErrorCode.FileTooLarge,
+      message: `File size exceeds the ${MAX_SIZE / 1000 / 1000}MB.`
+    }
+  }
+
+  return null
+}
+
+const useFiles = ({ maxFiles }: { maxFiles: number }) => {
+  const [files, setFiles] = useState<DropzoneFileUpload[]>([])
+  const [errorMessages, setError] = useState<string[]>([])
+  const [disabled, setDisabled] = useState<boolean>(false)
+
+  useEffect(() => {
+    if (files.length >= maxFiles) {
+      setDisabled(true)
+    } else {
+      setDisabled(false)
+    }
+  }, [maxFiles, files.length])
+
+  const addFiles: OnDropType = (acceptedFiles, rejectedFiles) => {
+    if (files.length + acceptedFiles.length + rejectedFiles.length > maxFiles) {
+      return setError(['Too many files'])
+    }
+    if (acceptedFiles.length > 0) {
+      const previousFiles = files
+      const newFiles = Array.from(acceptedFiles).map(file => ({
```

Comment:
Looks like it is already an array `File[]`
```suggestion
      const newFiles = acceptedFiles.map(file => ({
```